src/main/java/com/sonyericsson/jenkins/plugins/bfa/statistics/StatisticsLogger.java

```diff
@@ -126,7 +126,7 @@ public void run() {
             Node node = build.getBuiltOn();
             String nodeName = node.getNodeName();
             int timeZoneOffset = TimeZone.getDefault().getRawOffset();
-            String master = "";
+            String master;
```

Comment:
It's still a good practice to have the variable set to something over uninitialized. Who knows what will happen to the codes at line 140 in the future and may eventually turn out to produce uninitialized variable exception.

I would vote against the new code.